demo/src/main/java/com/orange/ods/demo/ui/components/appbars/top/AppBarTopCustomizationState.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ *
+ *  Copyright 2021 Orange
+ *
+ *  Use of this source code is governed by an MIT-style
+ *  license that can be found in the LICENSE file or at
+ *  https://opensource.org/licenses/MIT.
+ * /
+ */
+
+package com.orange.ods.demo.ui.components.appbars.top
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.saveable.rememberSaveable
+
+@Composable
+fun rememberAppBarTopCustomizationState(
+    navigationIconEnabled: MutableState<Boolean> = rememberSaveable { mutableStateOf(true) },
+    actionCount: MutableState<Int> = rememberSaveable { mutableStateOf(1) },
+    overflowMenuEnabled: MutableState<Boolean> = rememberSaveable { mutableStateOf(false) },
+) =
+    remember(navigationIconEnabled, actionCount, overflowMenuEnabled) {
+        AppBarTopCustomizationState(navigationIconEnabled, actionCount, overflowMenuEnabled)
+    }
+
+class AppBarTopCustomizationState(
+    val navigationIconEnabled: MutableState<Boolean>,
+    val actionCount: MutableState<Int>,
+    val overflowMenuEnabled: MutableState<Boolean>
+) {
+    val isNavigationIconEnabled: Boolean
+        get() = navigationIconEnabled.value
+
+    val isOverflowMenuEnabled: Boolean
+        get() = overflowMenuEnabled.value
+
+    val isOverflowMenuSwitchEnabled: Boolean
+        get() = actionCount.value <= ComponentAppBarsTop.MaxActionCount - 1
+
+    val maxActionCountSelectable: Int
```

Comment:
We could move `ComponentAppBarsTop.MinActionCount` and `ComponentAppBarsTop.MaxActionCount` here and make them private, rename this property to `maxActionCount` and add `minActionCount`. Then we could use `maxActionCount` and `minActionCount` in `ComponentAppBarsTop` (don't know if I'm clear 😅).